doc/src/manual/parallel-computing.md

```diff
@@ -1,10 +1,265 @@
 # Parallel Computing
 
-This part of the manual details the following types of parallel programming available in Julia.
+For any newcomer to multi-thread and parallel computing it can be useful to first appreciate 
+the different levels of parallelism offered by Julia. We can divide them in three main categories :
+
+1. Green Threading or Julia Coroutines 
+2. Multi-Threading
+3. Multi-Core or Distributed Processing
+
```

Comment:
It would be nice to have the rest of the page following these three points: i.e. have the H1 headings be equal to those three points.  I think that would mean to make below `## Scheduling` into `# Coroutines`, then rename `# Distributed Memory Parallelism` to `# Multi-Core  Processing` and move it to before `## Channels and RemoteChannels`, last make `## Multi-Threading` a level 1 header `# Multi-Threading`.  I haven't read the whole page, so maybe this simplistic scheme does not work, but if it does it would be clearer.

Reply:
Also in this scheme, `# Global Variables` would need to be demoted to a level 2 heading.

Reply:
Yes! I was also thinking to restructure it a bit, but I preferred to keep the paragraph's title to avoid to make it ttoo confusing.
Now the overall structure should look like  _:

# Parallel Computing
# Coroutines
    ##Channels
#Multi-Threading
    ##Setup
    ## The `@threads` Macro
    ## @threadcall (Experimental)
# Multi-Core or Distributed Processing
    ## Code Availability and Loading Packages
    ## Data Movement
    ## Global variables
    ## Parallel Map and Loops
    ## Remote References and AbstractChannels
    ## Channels and RemoteChannels
        ### Remote References and Distributed Garbage Collection
    ## [Shared Arrays]
        ### Shared Arrays and Distributed Garbage Collection
    ##ClusterManagers
        ### Cluster Managers with Custom Transports
        ### Network Requirements for LocalManager and SSHManager
        ### Cluster Cookie
    ## Specifying Network Topology (Experimental)
    ##Noteworthy external packages